Sources/QueuesRedisDriver/JobsRedisDriver.swift

```diff
@@ -55,6 +57,26 @@ public struct RedisQueuesDriver {
     ///   - eventLoopGroup: The `EventLoopGroup` to run the driver with
     public init(configuration config: RedisConfiguration, on eventLoopGroup: EventLoopGroup) {
         let logger = Logger(label: "codes.vapor.redis")
+        let eventLoop = eventLoopGroup.next()
```

Comment:
It's better to use `any()` than `next()` in modern NIO for better distribution across event loops.
```suggestion
        let eventLoop = eventLoopGroup.any()
```